doc/source/dev/roadmap.rst

```diff
@@ -79,13 +79,16 @@ Implement sparse arrays in addition to sparse matrices
 
 The sparse matrix formats are mostly feature-complete, however the main issue
 is that they act like ``numpy.matrix`` (which will be deprecated in NumPy at
-some point).  What we want is sparse *arrays* that act like ``numpy.ndarray``.
-This is being worked on in https://github.com/pydata/sparse, which is quite far
-along.  The tentative plan is:
-
-- Start depending on ``pydata/sparse`` once it's feature-complete enough (it
-  still needs a CSC/CSR equivalent) and okay performance-wise.
-- Indicate in the documentation that for new code users should prefer
-  ``pydata/sparse`` over sparse matrices.
-- When NumPy deprecates ``numpy.matrix``, vendor that or maintain it as a
-  stand-alone package.
+some point). What we want is sparse *arrays* that act like ``numpy.ndarray``.
+Sparse arrays have largely been implemented in scipy/sparse at this time. Some
+functionality is still being completed. The future plan is:
+
+- Provide a feature-complete sparse array api (including 1d-array).
+    - Extend sparse array api to 1d arrays:
+        - coo, csr and dok formats.
+        - the csr 1d format uses 2d csr sparsetools to do 1d things like
```

Comment:
style nit: could you use capitalization (COO, CSR, DOK) and 1-D/2-D here? Same higher up, "API" instead of "api" etc.

Reply:
`sparsetools` shouldn't be mentioned like this, since it's a private implementation detail. I think you can simply say "2-D CSR format to do 1-D ...".